tensorrec/tensorrec.py

```diff
@@ -351,10 +367,12 @@ def fit(self, interactions, user_features, item_features, epochs=100, learning_r
                          alpha=alpha,
                          verbose=verbose,
                          out_sample_interactions=out_sample_interactions,
-                         user_batch_size=user_batch_size)
+                         user_batch_size=user_batch_size,
+                         n_sampled_items=n_sampled_items)
 
     def fit_partial(self, interactions, user_features, item_features, epochs=1, learning_rate=0.1,
-                    alpha=0.00001, verbose=False, out_sample_interactions=None, user_batch_size=None):
+                    alpha=0.00001, verbose=False, out_sample_interactions=None, user_batch_size=None,
```

Comment:
pretty small default alpha?

Reply:
It is -- I wanted to have a small, non-zero alpha in place but because the relationship between weight normalization and the loss function is strong, I didn't want to have the alpha so high that it destroys performance for some loss functions.